NETDefault/UnitTestingWithNUnit&Moq/UnitTesting.md

```diff
@@ -1,6 +1,12 @@
 # Unit Testing with NUnit and Moq
 
-In this directory you will find a small project where we demonstrate the usage of NUnit and Moq to bring our code under test. Here, a _Service_ is being the subject under test, which depends on a _Gateway_ for completing the _GetDashboard_ call. Since we are unit testing, we do *not* want to include the external calls with the Gateway because that is not what we want to test and it would take far too long to complete. It is therefore that we create a fake instance of this gateway -  a.k.a. a mock, moq, substitute - for which we use [Moq](https://github.com/moq). Our _Service_ is the actual concrete class we want to test, which we do with [NUnit](https://github.com/moq).
+In this directory you will find a small project where we demonstrate the usage of NUnit and Moq to bring our code under test. 
+Here, a _Service_ is being the subject under test, which depends on a _Gateway_ for completing the _GetDashboard_ call. \
+Since we are unit testing, we do *not* want to include the external calls with the Gateway because that is not what we want to 
```

Comment:
I would suggest adding something of a Verify in this example, just as an demonstration on how it should be used. I know that you want to double down on only testing only the in-and-out of an interface, but let's be real, most of our services have lots of side effects, which cannot be verified by just verifying the output. It's not all just CRUD.

Sometimes you want to write a unit test that checks that something critical has happened, and which can only be verified by checking if the invocation on the mock happened. It's better to document how to do it, instead of not documented it. So let's verify whether some log line was written to a mocked `ILogger`.

Reply:
Ah, but that would indeed be a useful use case indeed. The current version only shows superfluous stuff so that is why I removed it.

Reply:
@ThomasBleijendaal added a test using ILogger and testing the error message is logged.